eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/io/FastaSequenceReader.java

```diff
@@ -148,6 +149,28 @@ public boolean doesContigExist(String contig) {
         return sequenceDictionary.getSequence(contig) != null;
     }
 
+    public ImmutablePair<String, Long> getContextNucleotide(String contig, long start, String reference,
```

Comment:
`getContextNucleotideAndNewStart`? it was not trivial to understand how and why the newstart should be used